### PR TITLE
Removed node_modules and updated .gitignore

### PR DESCRIPTION
**Pull request by anjannair**
I improved the `index.js` file by adding the _other random stuff_ to the main `client.on( 'message')....` function.

`.gitignore` was totally the copy of some xml app which isn't related to the discord bot

`node_modules` isn't required as it is automatically installed when a person runs it on their device. Its better to not include it in the source code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,95 +1 @@
...
-
-# SonarQube Plugin
-.idea/**/sonarIssues.xml
-
-# Markdown Navigator plugin
-.idea/**/markdown-navigator.xml
-.idea/**/markdown-navigator/
-
-# End of https://www.gitignore.io/api/intellij
+node_modules
```
Comment (singhayushh) [inline .gitignore:1]: You should never removed any thing from  .gitignore when making open source contributions without consent of the owner or if you're 100% sure that they aren't needed. 
The current configs that you removed were auto generated  .gitgjire for IntelliJ, which the owner very likely has been using

**Review by singhayushh — CHANGES_REQUESTED** [review]
Hello, please revert back the deletions you made from the  .gitignore as they are required when running intelliJ as text editor, which the author most likely has been doing. 

Also, I didn't understand why you replaced `===` with `==`. Both the operands are strings anyway

**Comment by anjannair** [comment]
Oh I did not know intelliJ wanted such a big `.gitignore` cause I have been using VScode and sublime text for quite some time. 

Usually when using node packages the `node_modules` is the only thing we include in `.gitignore`. Since he's open sourcing it, the general public **may** not use intelliJ.

Now coming to the difference between === and == 
The triple equal performs type casting for equality check. That is if you compare 2 and "2" it will give false
The double equal performs identify equality check. In this 2 and "2" will return true.

Link to an article that may explain [better](https://www.guru99.com/difference-equality-strict-operator-javascript.html#3)

It's much better to use the double equal.

**Review by singhayushh — APPROVED** [review]
It's all good now.

**Comment by KazMalhotra** [comment]
The reason _other random stuff_ is not in the main function is because the contents of it are not meant to be commands. They are meant to be responses to messages not beginning with the prefix '='. 

I suppose you could delete `node_modules`, from the repo, and make users do `npm install`, but I think it's fine. If you want to keep `node_modules` out of the repo, please add and installation section in the readme.md.

**Comment by anjannair** [comment]
> responses to YAGPDB
Ahh sorry about that. I'll fix that.

>but I think it's fine
There is a chance that the node modules you upload here may get outdated over time. So instead of uploading it the user can install it.

Yes I added the steps in the updated README and I changed the `index.js` to the old one with a few minor changes.

File: index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const Discord = require("discord.js");
 const client = new Discord.Client();
-const prefix = '=';
+const prefix = '='
```
Comment (singhayushh) [inline index.js]: ??

**Review by singhayushh — APPROVED** [review]
Approved!

File: index.js (hunk 2 above)
Comment (anjannair) [reply index.js]: I just left the `index.js` as it was before.
Semicolons don't matter.
I removed the brackets cause I just wanted it to look neater.

**Comment by anjannair** [comment]
Done changed the title too!

**Comment by KazMalhotra** [comment]
Looks good!

**Pull request merged**